experimental/tools/wheelmaker.py

```diff
@@ -102,6 +102,13 @@ def arcname_from(name):
 
             return normalized_arcname
 
+        if os.path.isdir(real_filename):
```

Comment:
I'd be curious to know your rationale for doing this here instead of in the starlark rules [here](https://github.com/bazelbuild/rules_python/blob/master/experimental/python/wheel.bzl#L104-L105) as @aaliddell discussed in the comments.

Reply:
I was about to say, I couldn't do it there.